### DISCUSSION TITLE
程序根本打不开怎么办

### DISCUSSION THREAD

**Discussion by no-good-name** [mikumifa/biliTickerBuy#80] [提问专区]
程序完全打不开，点击运行后直接就闪退了

**Comment by mikumifa** [comment]
贴一下日志，app.log

**Reply by no-good-name** [reply]
怎么弄啊，我纯电脑小白

**Comment by mikumifa** [comment]
你要给我具体的报错，截图之类的，不然我没办法告诉你哪有问题，exe程序同目录有一个app.log 把那个给我也行

**Comment by no-good-name** [comment]
![Uploading 1717593173900.jpg…]()

**Reply by mikumifa** [reply]
没发出来

**Comment by no-good-name** [comment]
图片发不出来，目录下没找到app.log，程序是运行后有一个弹窗然后马上关闭了

**Comment by mikumifa** [comment]
检查一下有没有下载Edge浏览器，
然后看看这个 
 https://github.com/mikumifa/biliTickerBuy/wiki/%E6%B5%8F%E8%A7%88%E5%99%A8%E9%A9%B1%E5%8A%A8%E5%AE%89%E8%A3%85%E5%87%BA%E9%97%AE%E9%A2%98%E7%9A%84%E8%A7%A3%E5%86%B3%E6%96%B9%E5%BC%8F

**Comment by no-good-name** [comment]
大佬，问一下应用程序无法正常启动(0x000007b)怎么解决

**Comment by mikumifa** [comment]
就给怎么错误码没办法定位，你先试一下上面那个链接

**Reply by no-good-name** [reply]
试了，打不开

**Reply by mikumifa** [reply]
https://www.puresys.net/5055.html 试试这个，

**Comment by Andy-Chuck** [comment]
什么操作系统
